scripts/secrets.sh

```diff
@@ -20,6 +20,8 @@ function main {
     echo "export $secret" >> $env_secret_file
   done
 
+  echo "export CONFIGCAT_SDK_KEY='$(aws ssm --region eu-west-1 get-parameter --name=/k8s/configcat/sdk-key --profile=island-dev --with-decryption --query=Parameter.Value --output=text)'" >> $env_secret_file
```

Comment:
This should not be needed. We can fetch this variable with:
`./secrets.sh configcat`

Would be nicer to rename the parameter in the parameter store as: `CONFIGCAT_SDK_KEY`

Reply:
Just needs the `dev` label (I added it on dev, but as I mentioned above, it needs a rename)

Reply:
That would override the secrets for the service you're developing. This environment variable is set for all services, always, so I felt it made more sense to include it for all services. That way you can keep doing `./scripts/secrets.sh gjafakort` (see https://github.com/island-is/helm/pull/275)

Reply:
No it doesn't. It only overrides if you pass the --rest flag with. 

So this would just safely append it to the list

Reply:
![crawlintohole](https://user-images.githubusercontent.com/683742/106715651-47be9f80-65f5-11eb-9efc-82f060ac69a4.gif)

Reply:
![notthedroids](https://user-images.githubusercontent.com/683742/106715667-4d1bea00-65f5-11eb-9081-edea816b60e6.gif)